SDKAndroid/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
     compileSdk 32
 
     defaultConfig {
-        minSdk 23
+        minSdk 26
```

Comment:
Why did we increase this?

Reply:
Cause SDK version 23 is super old, is Android Is the Android minimal version, but this version is super old, this versions is Android M (Marshmallow) released in 2015, but Google annaunces that the minimal support for incoming Compose tools gonna be 26, for that reason im increase the level.

And second thing is because with this new SDK level i can reduce the use of @ExperimentalAPI in Compose, reducing the compile time.